src/pages/SearchInput/searchInput.tsx

```diff
@@ -63,10 +69,9 @@ export const SearchInput = () => {
                     <input
                         value={query}
                         type="text"
-                        placeholder="Search..."
+                        placeholder="🔎 Search..."
```

Comment:
please remove this icon from placeholder, if you need this kind of icon please add styled "::before"

Reply:
@szu-szy